utils/enums/card.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+
+from utils.date_util import *
+from utils.enums.card_type import CardType
+
+
+class Card(Enum):
+    def __init__(self, formatted_number, expiration_date: str, card_type: CardType, cvv):
+        self.formatted_number = formatted_number
+        self.expiration_date = expiration_date
+        self.type = card_type.name
+        self.cvv = cvv
+
+    # expiration date may be fixed value but if someone prefers more "dynamic" approach
+    # there are 2 methods: future_expiration_date and past_expiration_date
+
+    AMEX = '3400 000000 00611', '12/23', CardType.AMEX, 1234
```

Comment:
I did not think a lot about names for this cards. We will figure it out by using those in real cases.
Thin about it as the example/template